### PR TITLE
[RUM-5705] Collect Long Animation Frames

### PR DESCRIPTION
**Pull request by thomas-lebeau**
## Motivation

Collect Long animation frame events instead of long task. 

## Testing

- [ ] Local
- [ ] Staging
- [ ] Unit
- [ ] End to end

---

I have gone over the [contributing](https://github.com/DataDog/browser-sdk/blob/main/CONTRIBUTING.md) documentation.

### CODE REVIEW

File: packages/rum-core/src/domain/longAnimationFrame/longAnimationFrameCollection.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+      const startClocks = relativeToClocks(entry.startTime)
+      const rawRumEvent: RawRumLongAnimationFrameEvent = {
+        date: startClocks.timeStamp,
+        long_task: {
+          id: generateUUID(),
+          entry_type: RumLongTaskEntryType.LONG_ANIMATION_FRAME,
+          duration: toServerDuration(entry.duration),
+          blocking_duration: toServerDuration(entry.blockingDuration),
+          first_ui_event_timestamp: relativeToClocks(entry.firstUIEventTimestamp).relative,
```
Comment (thomas-lebeau) [inline packages/rum-core/src/domain/longAnimationFrame/longAnimationFrameCollection.ts:27]: Despites the name suggest, it's a relative time similarly to `renderStart`

**Review by N-Boutaib — APPROVED** [review]
Don't we need to update `LongTask` event type in `rum-event-format` first 🤔 ?

**Review by nulrich — APPROVED** [review]

**Comment by blackswanny** [comment]
@thomas-lebeau @nulrich  hi, I am looking for this feature. PerformanceLongAnimationFrameTiming or PerformanceScriptTiming is what shows more information

Currently I see "Long tasks" duration in Datadog RUM Explorer (app.datadoghq.com/rum/sessions) , but no more information , such as script source, function, etc, which is not helpful for debugging.
We use v5.26.0 npm for rum package, but I still don't see new facets in RUM Explorer. Do you know how to make it work?

**Comment by thomas-lebeau** [comment]
Hi @blackswanny, Thanks for your interest, unfortunately this is not available yet as we're still working on polishing the UI experience.
I'll let you know when this is available!

**Pull request merged**